nettacker/core/arg_parser.py

```
@@ -23,6 +24,57 @@
 
 log = get_logger()
 
+# Regex pattern for port validation: single port (123) or range (80-90)
+PORT_PATTERN = re.compile(r"^\d+(-\d+)?$")
+
+
+def validate_and_parse_ports(port_string, error_prefix=""):
```

Ruff (0.14.14)

[warning] Unused function argument: `error_prefix`

(ARG001)